tools/changelog-generator/docs_changelog.md

```diff
@@ -0,0 +1,170 @@
+## 2023-07-12 - 2023-07-19
+
+### PR #5802: [feat(mesh): inherit sidebar from Kuma](https://github.com/Kong/docs.konghq.com/pull/5802)
+
+**Updates**
+
+- [app/_src/.repos/kuma](https://docs.konghq.com/.repos/latest/kuma)
+- [app/_src/mesh/features/acmpca.md](https://docs.konghq.com/mesh/latest/features/acmpca)
+- [app/_src/mesh/features/cert-manager.md](https://docs.konghq.com/mesh/latest/features/cert-manager)
+- [app/_src/mesh/features/kds-auth.md](https://docs.konghq.com/mesh/latest/features/kds-auth)
+- [app/_src/mesh/features/opa.md](https://docs.konghq.com/mesh/latest/features/opa)
+- [app/_src/mesh/features/vault.md](https://docs.konghq.com/mesh/latest/features/vault)
+- [app/_src/mesh/features/windows.md](https://docs.konghq.com/mesh/latest/features/windows)
+- [app/_src/mesh/gettingstarted.md](https://docs.konghq.com/mesh/latest/gettingstarted)
+- [app/_src/mesh/index.md](https://docs.konghq.com/mesh/latest/index)
+- [app/_src/mesh/installation/docker.md](https://docs.konghq.com/mesh/latest/installation/docker)
+- [app/_src/mesh/installation/ecs.md](https://docs.konghq.com/mesh/latest/installation/ecs)
+- [app/_src/mesh/installation/helm.md](https://docs.konghq.com/mesh/latest/installation/helm)
+- [app/_src/mesh/installation/kubernetes.md](https://docs.konghq.com/mesh/latest/installation/kubernetes)
+- [app/_src/mesh/installation/openshift.md](https://docs.konghq.com/mesh/latest/installation/openshift)
+- [app/_src/mesh/installation/windows.md](https://docs.konghq.com/mesh/latest/installation/windows)
+
+---
+
+### PR #5822: [Add data-panel attribute to navtabs](https://github.com/Kong/docs.konghq.com/pull/5822)
```

Comment:
Q: Are you purposely avoiding linking to any files outside of the `app/_src` directory? If so, why?